src/App.jsx

```diff
@@ -1,10 +1,55 @@
-import React from 'react';
-import './App.css';
+import { React, useState } from 'react';
+import axios from 'axios';
+
+const geocodingURI = 'http://api.openweathermap.org/geo/1.0/direct?q=';
+const weatherApiKey = process.env.REACT_APP_WEATHER_API_KEY;
 
 function App() {
+  const [geoData, setGeoData] = useState({});
+
+  const retrieveGeoData = async (city = '') => {
```

Comment:
We could error check with input validation, since there are all kinds of potential errors with city name input.  

That does bring up the question of where errors should be handled within the various API calls we're making, since I assume the API calls taking `retrieveGeoData` will have to account for those errors too.  

It's also something we can discuss further in chat/meetings!

Reply:
@awong-pdx I think the error handling for the API calls should be done right where it is being implemented instead of the client side. You're right, retrieveGeoData will return an error but then component or the client that depends on the coordinates from this function will then need to handle that error has well. I think if I return a generic error, then the weatherRequest or whoever is calling it can check for that specific error message.

Reply:
Okay, that would be helpful to have a generic error to work with.